compositor_common/src/renderer_spec.rs

```diff
@@ -13,6 +13,24 @@ impl Display for RendererId {
     }
 }
 
+#[derive(Debug, Serialize, Deserialize, Clone, Copy)]
+#[serde(rename_all = "snake_case")]
+pub enum FallbackStrategy {
+    NeverFallback,
+    FallbackIfAllInputsMissing,
+    FallbackIfAnyInputsMissing,
+}
+
+impl FallbackStrategy {
+    pub fn default_shader_fallback_strategy() -> FallbackStrategy {
+        FallbackStrategy::FallbackIfAllInputsMissing
+    }
+
+    pub fn default_shader_web_renderer_strategy() -> FallbackStrategy {
+        FallbackStrategy::FallbackIfAllInputsMissing
+    }
+}
```

Comment:
```suggestion
fn default_shader_fallback_strategy() -> FallbackStrategy {
    FallbackStrategy::FallbackIfAllInputsMissing
}

fn default_shader_web_renderer_strategy() -> FallbackStrategy {
    FallbackStrategy::FallbackIfAllInputsMissing
}
```
I assume serder does not need those methods to be public?